packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -239,6 +246,62 @@ class ToolbarSpacers extends React.Component {
 }
 ```
 
+### Sticky
+
+```js
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarContent } from '@patternfly/react-core';
+import { Button, ButtonVariant, InputGroup, TextInput, Checkbox } from '@patternfly/react-core';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+
+const ToolbarItems = () => {
+  const [isSticky, setIsSticky] = React.useState(true);
+  const [showEvenOnly, setShowEvenOnly] = React.useState(true);
+  const array = [...Array(25).keys()];
+  const numbers = showEvenOnly ? array : array.filter(number => number % 2 === 0);
```

Comment:
I think you've got this ternary reversed, shouldn't it be returning the array unchanged when `showEvenOnly` is false?

Reply:
oops, fixed. thanks!